screengrab/lib/screengrab/runner.rb

```diff
@@ -225,7 +225,11 @@ def run_tests(device_serial, test_classes_to_use, test_packages_to_use)
                                       print_all: true,
                                       print_command: true)
 
-        UI.user_error!("Tests failed", show_github_issues: false) if test_output.include?("FAILURES!!!")
+        if @config[:exit_on_test_failure]
```

Comment:
I would wrap this block in the `if test_output.include?("FAILURES!!!")` but rubocop then complains I need to include a `next` to skip the current iteration of the loop (which is not necessary)